README.md

```diff
@@ -1 +1,35 @@
-# hello-microverse
\ No newline at end of file
+<html>
+<h1>Hello Microverse</h1>
+<br>
+<h2>Hello Microverse project</h2>
+<br>
+<h3>Built With</h3>
+<br>
+<ul>
+    <li>Webhint</li>
+    <li>Styleint </li>
+    <li>HTML5</li>
+    <LI>CSS3</LI>
+</ul>
+
+
+<h2><Authors/h2>
+<p>👤 Author1</p>
+
+<ul>
+   <li>GitHub: @andreacostabz</li> 
+    <li>Twitter: @andrebze</li>
+   <li>LinkedIn: Andre Acosta</li>
```

Comment:
[OPTIONAL]
- A Good way of GitHub, Twitter & Linkedin is by providing links beside it. for example 👇 
`GitHub: [@githubhandle](https://github.com/Username)`

Reply:
awesome!! Thank you!